common/action-manager.js

```diff
@@ -19,7 +19,6 @@ const am = new ActionManager([
   "EXPERIMENTS_RESPONSE",
   "NOTIFY_BLOCK_URL",
   "NOTIFY_UNBLOCK_URL",
-  "NOTIFY_UNBLOCK_ALL",
```

Comment:
I think the idea was to add this to the dev page, could you maybe file a ticket for that?